package.json

```diff
@@ -58,44 +58,44 @@
     "@adobe/helix-status": "8.1.3",
     "@adobe/openwhisk-action-logger": "2.2.0",
     "@adobe/openwhisk-action-utils": "4.3.0",
-    "@emmetio/expand-abbreviation": "^0.7.3",
-    "ajv": "^6.10.2",
-    "callsites": "^3.1.0",
-    "clone": "^2.1.2",
-    "dompurify": "2.0.14",
-    "dot-prop": "^5.2.0",
-    "ferrum": "^1.2.0",
-    "fs-extra": "^9.0.0",
-    "github-slugger": "^1.2.1",
-    "hast-util-find-and-replace": "^3.0.0",
-    "hast-util-from-dom": "^3.0.0",
-    "js-yaml": "^3.13.1",
+    "@emmetio/expand-abbreviation": "0.7.3",
+    "ajv": "6.12.4",
+    "callsites": "3.1.0",
+    "clone": "2.1.2",
+    "dompurify": "2.0.15",
+    "dot-prop": "5.2.0",
+    "ferrum": "1.7.0",
+    "fs-extra": "9.0.1",
+    "github-slugger": "1.3.0",
+    "hast-util-find-and-replace": "3.1.0",
+    "hast-util-from-dom": "3.0.0",
+    "js-yaml": "3.14.0",
     "jsdom": "16.4.0",
-    "lodash": "^4.17.15",
-    "mdast-util-to-hast": "^9.0.0",
-    "mdast-util-to-string": "^1.0.6",
-    "micromatch": "^4.0.2",
-    "object-hash": "^2.0.0",
-    "path-to-regexp": "^6.1.0",
-    "property-information": "^5.2.2",
+    "lodash": "4.17.20",
```

Comment:
do we need lodash? maybe we can replace it with only the package of the function that we need

Reply:
We need lodash in several places.